src/schema/city/index.ts

```diff
@@ -115,6 +117,35 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
           artGuideUrl: {
             type: GraphQLString,
           },
+          featuredShows: {
+            type: new GraphQLList(ShowType),
+            resolve: (citySponsoredContent, _args, { showsLoader }) => {
+              let gravityOptions = {
+                id: citySponsoredContent.showIds,
+                include_local_discovery: true,
+                status: "current",
+                sort: "end_at",
+                size: 2,
```

Comment:
Here's the canned query that captures our definition of "featured shows" so that Emission never has to bother with it. 

🤷‍♂️ ?